### PR TITLE
Show certificate - url, crt file, keyfile

### PR DESCRIPTION
**Pull request by uli-heller**
* Added ability to show contents of crt and key files
* Added md5 hash of the key

------

**I have checked the following requirements before making this PR.**

- [x] The script is marked as executable (`chmod +x`).
- [x] I have added/updated the README entry for the script (if applicable).
- [x] I have mentioned any outstanding dependencies and OS requirements for the script in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,13 +168,14 @@ rm-ds-store
...
 
-Script: [show-certificate](show-certificate)  
-Dependencies: [openssl](https://www.openssl.org/)
+Script: [show-certificate](show-certificate)
 
 ```sh
 show-certificate google.com
 ```
 
+Depends on openssl.
```
Comment (aviaryan) [inline README.md]: Please revert this change with dependencies as that's how dependencies are listed everywhere.

**Review by aviaryan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (uli-heller) [reply README.md]: Done

**Review by aviaryan — APPROVED** [review]
LGTM
